prog/bootstrap_rhizome.rb

```diff
@@ -10,7 +10,7 @@ def user
   end
 
   label def start
-    sshable.update(raw_private_key_1: SshKey.generate.keypair)
+    sshable.update(raw_private_key_1: SshKey.generate.keypair) if sshable.raw_private_key_1.nil?
```

Comment:
```suggestion
    sshable.this.where(raw_private_key_1: nil).update(raw_private_key_1: SshKey.generate.keypair)
```

Reply:
actually I guess this makes the testing much harder, so nevermind, unless you think it's a good idea for some reason.